crates/polars-core/src/chunked_array/logical/categorical/mod.rs

```diff
@@ -57,6 +57,21 @@ impl CategoricalChunked {
         &mut self.logical
     }
 
+    pub(crate) fn get_flags(&self) -> u8 {
+        self.bit_settings.bits
+    }
+
+    /// Set flags for the Chunked Array
+    ///
+    /// # Safety
+    /// The caller must ensure the flags are correct for the underlying chunks
+    pub(crate) unsafe fn set_flags(&mut self, flags: u8) -> PolarsResult<()> {
+        let settings = BitSettings::from_bits(flags)
+            .ok_or(polars_err!(ComputeError : "Corrupt flags {} for {}",flags,self.dtype()))?;
```

Comment:
We try to have no capital letters in single line errors.

```suggestion
            .ok_or(polars_err!(ComputeError : "corrupt flags {} for {}",flags,self.dtype()))?;
```